Modules/atexitmodule.c

```diff
@@ -278,11 +278,20 @@ atexit_unregister(PyObject *module, PyObject *func)
             continue;
         }
 
+        /*
+         * Increment refcounts of func and cb->func because equality check may
+         * unregister one or both
+         */
+        PyObject *cb_func = Py_NewRef(cb->func);
+        PyObject *cur_func = Py_NewRef(func);
```

Comment:
`func` is already a strong reference here; there is no need to incref it one more time.